### ISSUE TITLE
-commit-mode and --tagged not working together

### ISSUE THREAD

**Issue by orSpec** [instaloader/instaloader#483]
If downloading with flags --commit-mode and --tagged, it seems that it is never checked if the post was already downloaded. 

Normally, it is checked if the json exists. This check seems to be missing when downloading tagged posts or is not working properly.

**Comment by aandergr** [comment]
Thanks for reporting! Related to #257, in that it refers to the broken `--commit-mode`.

**Comment by aandergr** [comment]
This has been fixed by #697.

> Rather than checking the json file to make sure posts have been successfully downloaded, data is stored in a temporary file which is renamed when downloading has finished.

It is merged into `upcoming/v4.5`, which can be tried out with:

```
pip3 install --upgrade git+https://github.com/instaloader/instaloader@upcoming/v4.5
```
